<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>helpWindow</class>
 <widget class="QWidget" name="helpWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="VerticalLayout">
    <item>
     <layout class="QHBoxLayout" name="HorizontalLayout">
      <item>
       <widget class="QPushButton" name="BackButton">
        <property name="text">
         <string>Назад</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="HomeButton">
        <property name="text">
         <string>Домой</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="NextButton">
        <property name="text">
         <string>Вперёд</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextBrowser" name="TextBrowser">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>9999999</width>
        <height>9999999</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
